skills/_skill_common.py

```
@@ -0,0 +1,46 @@
+    home = os.environ.get("DATUS_HOME", "")
+    namespace = os.environ.get("DATUS_NAMESPACE", "")
+
+    if not config_path:
+        print(json.dumps({"success": False, "error": "DATUS_CONFIG_PATH environment variable not set"}))
+        sys.exit(1)
+
+    try:
+        from datus.configuration.agent_config_loader import load_agent_config
+
+        agent_config = load_agent_config(config=config_path)
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)